mcmd/commands/set.py

```diff
@@ -34,6 +34,11 @@ def add_arguments(subparsers):
                        type=str,
                        help="The value to set the attribute to")
 
+    p_set.add_argument('--for-id', '-i',
```

Comment:
I think I prefer `--for-entity` or `--for-row` here because you're not actually setting the value for the ID, but for the row with a certain ID. Matter of taste maybe, I'll let you decide ;)